src/css/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+@mixin flexBox ($direction, $justify: none, $align: none){
+  display: flex;
+  flex-direction: $direction;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+@mixin onePercentMargin {
+  margin: 1%;
+}
+
+@mixin buttons {
+  cursor: pointer;
+  border: none;
+}
+
+@mixin backgroundImages ($url) {
+  background-image: url($url);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+@mixin standardBorderRadius {
```

Comment:
I think this could maybe be a variable too. Unless you want the option to add more properties later.